Jenkinsfile

```diff
@@ -3,6 +3,6 @@ buildJavaProject(
     buildInfo: getBuildInfo(),
     integrationTestsEnvVars: [],
     shouldPublishJars: true,
-    shouldPublishDockerImages: false,
-    notifyJobs: ['iexec-core', 'iexec-worker']
+    shouldPublishDockerImages: false
+    //notifyJobs: ['iexec-core', 'iexec-worker']
```

Comment:
```suggestion
```

Reply:
I asked myself if we wanted to keep the comment or not.
I removed it in 03c4917